docs/fileobject.rst

```diff
@@ -293,9 +293,13 @@ Version methods
 
 .. method:: version_name(version_suffix, extra_options=None)
 
-    :param version_suffix: A version name as defined in :ref:`settingsversions_versions`.
-    :param extra_options: Optional ``dict`` that will be merged with the
-        predefined ``VERSION`` definition if the given ``version_suffix`` exists.
+    :param version_suffix: A suffix to compose the version name acordinly to
+        the :ref:`settingsversions_version_namer` in use. If the suffix
+        is a key in :ref:`settingsversions_versions` the options will be used.
```

Comment:
a) accordingly
b) what if the suffix is NOT a key in VERSIONS
c) with options you probably mean extra_options, right?